action-app_template.py

```diff
@@ -32,6 +36,24 @@ def __init__(self):
         # start listening to MQTT
         self.start_blocking()
 
+    @staticmethod
+    def mqtt_options():
+        snips_commons = SnipsConfigParser.read_configuration_file(PLATFORM_CONFIG_INI).get('snips-commons', {})
+        if not snips_commons:
+            snips_commons = SnipsConfigParser.read_configuration_file(CONFIG_INI).get('snips-commons', {})
+
+        return MqttOptions(
+            broker_address=snips_commons.get('mqtt') or f'{MQTT_IP_ADDR}:{MQTT_PORT}',
+            username=snips_commons.get('mqtt_username') or None,
+            password=snips_commons.get('mqtt_password') or None,
+            tls_hostname=snips_commons.get('mqtt_tls_hostname') or None,
+            tls_ca_file=snips_commons.get('mqtt_tls_cafile') or None,
+            tls_ca_path=snips_commons.get('mqtt_tls_capath') or None,
+            tls_client_key=snips_commons.get('mqtt_tls_client_key') or None,
+            tls_client_cert=snips_commons.get('mqtt_tls_client_cert') or None,
+            tls_disable_root_store=snips_commons.get('mqtt_tls_disable_root_store') or False
+        )
+
```

Comment:
add support for auth and non default broker address (check snips.toml->config.ini->use default)